src/components/Card.js

```diff
@@ -1,39 +1,24 @@
 import FaveBtn from "./FaveBtn";
 import styled from "styled-components";
 
-export default function Card({
-  id,
-  image,
-  title,
-  servings,
-  duration,
-  description,
-  favorite,
-  recipes,
-  setRecipes,
-}) {
+export default function Card({ recipe }) {
   return (
     <StyledCard>
-      <FaveBtn
-        id={id}
-        favorite={favorite}
-        recipes={recipes}
-        setRecipes={setRecipes}
-      />
-      <img src={image} alt={title} />
-      <h3>{title}</h3>
+      <FaveBtn id={recipe.id} isFavorite={recipe.isFavorite} />
+      <img src={recipe.image} alt={recipe.title} />
+      <h3>{recipe.title}</h3>
```

Comment:
instead of writing everytime `recipe.` you could destructure the `recipe` object after line 4

```js
export default function Card({ recipe }) {
const {id, isFavorite, title } = reccipe
//...
```

and then just use `id` and so so.

Reply:
AHA noted! Thanks Martin! :)